ubuntu-checkpatch

```diff
@@ -362,8 +365,24 @@ class CheckCherrypickBackport(Check):
             commit = self.patch.backport
             self.result(PASS, "valid backport from commit: " + commit)
 
-        # TODO: Check provenance and upstream commit existence
-        return 0
+        if commit and self.patch.provenance in ("", "linux-next", "linux-stable"):
+            linux_base = "https://git.kernel.org/pub/scm/linux/kernel/git"
+            upstream = {
+                "": f"{linux_base}/torvalds/linux.git/commit/?id={commit}",
```

Comment:
Can we stick to format() instead of f-string? This script might be run on older releases with an older Python.

Reply:
yes, done.